pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go

```diff
@@ -81,7 +82,13 @@ func (tr *TrustedTaskRun) verifyTaskRun(
 	k8sclient kubernetes.Interface,
 	tektonClient versioned.Interface,
 ) (errs *apis.FieldError) {
+	trustedConfig := trustedconfig.FromContextOrDefaults(ctx)
 	logger := logging.FromContext(ctx)
+	if trustedConfig.SkipValidation {
+		logger.Info("Skipping TaskRun verification.")
+		return nil
+	}
```

Comment:
Sorry my bad. I realized that it may doesn't make sense to directly skip it, since we can remove the webhook to do the same thing. (But configmap is more convenient though)

Maybe we should support these:
1) Config is true: If fail the verification, we add one field in annotation, logger the error information and return nil.
2) Config is false: Return err.